packages/core/src/generators/imports.ts

```diff
@@ -234,6 +234,7 @@ export const addDependency = ({
 
       if (types) {
         if (values) {
+          types = types.filter((t) => !values.some((v) => v.name === t.name));
```

Comment:
You should avoid overriding variable names whenever possible, so give them different names 👍 

```suggestion
          uniqueTypes = types.filter((t) => !values.some((v) => v.name === t.name));
```